modules/ting/ting.module

```diff
@@ -82,7 +82,7 @@ function ting_menu() {
     'position' => 'left',
     'weight' => 20,
     'page callback' => 'system_admin_menu_block_page',
-    'access arguments' => array('access administration pages'),
+    'access arguments' => array('administer ting settings'),
```

Comment:
Will this require an change in ding_permissions module as well?

Reply:
Added the permission to the administrative roles.